documentation/api/io.strimzi.api.kafka.model.connect.KafkaConnectSpec.adoc

```diff
@@ -92,19 +92,15 @@ Use a curl request to get a complete list of Kafka Connect loggers running from
 curl -s http://<connect-cluster-name>-connect-api:8083/admin/loggers/
 ----
 
-Kafka Connect uses the Apache `log4j` logger implementation.
-
+Kafka Connect uses the Apache `log4j2` logger implementation.
 Use the `logging` property to configure loggers and logger levels.
 
-You can set the log levels by specifying the logger and level directly (inline) or use a custom (external) ConfigMap.
-If a ConfigMap is used, you set `logging.valueFrom.configMapKeyRef.name` property to the name of the ConfigMap containing the external logging configuration. Inside the ConfigMap, the logging configuration is described using `log4j.properties`. Both `logging.valueFrom.configMapKeyRef.name` and `logging.valueFrom.configMapKeyRef.key` properties are mandatory. A ConfigMap using the exact logging configuration specified is created with the custom resource when the Cluster Operator is running, then recreated after each reconciliation. If you do not specify a custom ConfigMap, default logging settings are used. If a specific logger value is not set, upper-level logger settings are inherited for that logger.
-For more information about log levels, see {ApacheLoggers}.
+NOTE: For Strimzi 0.45 and earlier, log4j1 was used for Kafka components. 
+Refer to the Strimzi 0.45 documentation for loggers and logging configuration examples based on log4j1.
```

Comment:
This is not a question of Strimzi version but Kafka version. So this needs ot be reworked here as well as in the other places. E.g.:

```suggestion
NOTE: Kafka 3.9 and earlier uses Log4j1 got logging. 
Refer to the Strimzi 0.45 documentation for loggers and logging configuration examples based on log4j1 that you can use with Strimzi 0.46 and Kafka 3.9.x: <LINK-TO-THE-DOCS>
```

**We should probably also mark it as IMPORTANT and not NOTE and move it to the top of the section. Ir is pretty lost here in the middle and for users it makes zero sense to start reading this chapter if they use Kafka 3.9 as none of it is valid for them.**